banzai/tests/test_need_to_process_image.py

```diff
@@ -22,7 +17,11 @@ def __init__(self, success=False, checksum=md5_hash1, tries=0):
 @mock.patch('banzai.utils.file_utils.get_md5')
 @mock.patch('banzai.dbs.get_processed_image')
 @mock.patch('banzai.realtime.get_primary_header')
-def test_no_processing_if_previous_success(mock_header, mock_processed, mock_md5):
+@mock.patch('banzai.dbs.get_instrument')
+@mock.patch('banzai.utils.image_utils.image_can_be_processed')
+def test_no_processing_if_previous_success(mock_can_process, mock_instrument, mock_header, mock_processed, mock_md5):
+    mock_can_process.return_value = True
+    mock_instrument.return_value = FakeInstrument()
     mock_processed.return_value = FakeRealtimeImage(success=True, checksum=md5_hash1)
     mock_md5.return_value = md5_hash1
     assert not need_to_process_image('test.fits', FakeContext())
```

Comment:
I think some of these tests need to be update to pass `ignore_schedulability` instead of a `PipelineContext` instance to `need_to_process_image`. Should we be worried that they're passing anyway?